docs/developer-guide/infrastructure.md

```diff
@@ -6,5 +6,167 @@ DKMS infrastructure is composed of two main components: Witnesses and Watchers.
 - [ Watcher ](https://ghcr.io/thclab/keriox-watcher)
 
 For sample setup, see [this](https://github.com/THCLab/dkms-demo/blob/main/infrastructure/docker-compose.yml).
+and for more information see [setup](#setup) section.
 
-The number of nodes in the network is use-case dependent, but it is vital to analyze the system's tolerance to Byzantine failures. In the presence of Byzantine failures, the network must tolerate up to 1/3 of the malicious nodes. For this to be achievable, the total number of nodes `N` must be at least three times the number of malicious nodes plus one. In other words, `N = 3f +1`, where `f` is the amount of malicious nodes.
+## Argo
+
+Argo is a sandbox which allows you to test right away the DKMS infrastructure without need to setup anything.
+Checkout our public [dashboard](http://dashboard.argo.colossi.network/public-dashboards/2bdeda88c701459e8ec25802f79b7da2) to see the current state of the network.
+
+<iframe src="http://dashboard.argo.colossi.network/public-dashboards/2bdeda88c701459e8ec25802f79b7da2" width="100%" height="600" frameborder="0"></iframe>
+
+
+## Setup
+
+If you would like to run your own DKMS network, follow the instructions below.
+
+### Design your network
```

Comment:
The topic is far more complex and requires broader analysis that is out of scope for this documentation. Would change (and move somewhere below) this part and merely stress that:
- KERIOX impl brings infrastructure that demands consensus, and this consensus is achieved via the KAWA algorithm. 
- KAWA algorithm ensures immunity, that is, one agreement or none at all. 
- For a given $N=3F+1$ total amount of nodes, one seeks for $M=\frac{N+F+1}{2}$ that is the minimum M that satisfies the immunity. Note that for a F=1, N and M may be higher values, i.e., N=8 , then M=5. More N means adding HA.

Reply:
Let's create proper section for it.